ceph/cephfs/auth/openshift-clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: cephfs-provisioner-runner
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "create", "delete"]
```

Comment:
> I'm saying that data model doesn't make sense. It exposes details of how the storage backing the PVC to the user in the PVC namespace. At best, that means they can disrupt the operation by deleting the secret. At worst, it means admin credentials are exposed to namespaced users.

The admin credentials are never exposed to PV or PVC.  What is in the PV is the unprivileged user's credential.